tests/functional/lib.sh

```diff
@@ -58,19 +58,28 @@ force_cleanup_libvirt_disks() {
 }
 
 teardown() {
-    teardown_vagrant
-    force_cleanup_libvirt_disks
+    if [[ x"$HEKETI_TEST_NO_VAGRANT" != "xyes" ]]
```

Comment:
I don't think you need to do the traditional x prefix thing when using the bash double-bracket [[ test. I think the bash specific builtin is smart enough to do the comparision like a "normal" programming language.

Reply:
done